src/sys/socket/sockopt.rs

```diff
@@ -372,6 +372,16 @@ sockopt_impl!(
     libc::SO_LINGER,
     libc::linger
 );
+#[cfg(apple_targets)]
+sockopt_impl!(
+    /// `SO_LINGER_SEC` on apple targets is the genuine equivalent to
+    /// other platforms `SO_LINGER`. Indeed, the latter uses ticks.
```

Comment:
> is the genuine equivalent to other platforms `SO_LINGER`. 

This makes me feel that macOS does not provide `SO_LINGER` but only `SO_LINGER_SEC`, can we just say something like:

> Same as `SO_LINGETR`, but the duration is in seconds rather than kernel ticks.

Reply:
> 
> This makes me feel that macOS does not provide `SO_LINGER` but only `SO_LINGER_SEC`

oh apple does provide it (see my commit message, that s the C header snapshot).

, can we just say something like:
> 
> > Same as `SO_LINGETR`, but the duration is in seconds rather than kernel ticks.

Sure.